common/src/IO/FgdParser.cpp

```diff
@@ -52,7 +52,7 @@ FgdTokenizer::FgdTokenizer(const std::string_view str)
 {
 }
 
-const auto FgdTokenizer::WordDelims = std::string{" \t\n\r()[]?;:,="};
+const std::string FgdTokenizer::WordDelims = std::string{" \t\n\r()[]?;:,="};
```

Comment:
Why this change?

Reply:
MSVC and GCC don't like `auto`: https://gcc.godbolt.org/z/fTWP1b5oj